lesson15/js/main.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+const openModalBtn = document.getElementById('js-openModalBtn');
+openModalBtn.addEventListener(
+	'click',
+	() => {
+		const getModal = document.getElementById('js-modalContainer');
+		getModal.classList.add('is-visible');
+		const modalBg = document.createElement('div');
+		modalBg.id = 'js-bgFixed';
+		modalBg.classList.add('el_modalBg');
+		document.body.appendChild(modalBg);
+		document.body.style.position = 'fixed';
+		closeButton();
+	},
+	false
+);
+
+function closeModal() {
+	const getModal = document.getElementById('js-modalContainer');
+	getModal.classList.remove('is-visible');
+	const bgFixed = document.getElementById('js-bgFixed');
+	document.body.removeAttribute('style');
+	bgFixed.remove();
+}
```

Comment:
Not every time, but An error occurs when clicking the close button in the modal.
[![Image from Gyazo](https://i.gyazo.com/1bcddfc6618f724c94600135d76676ad.png)](https://gyazo.com/1bcddfc6618f724c94600135d76676ad)

My guess is that the click event is not well defined in the right place.
Please check.

Reply:
Thank you for review!!!!!!

I was able to understand the error by the debugging method you taught me.
I was able to fix the event of the close button and resolve the error.
Thank you!

[1ee8e03](https://github.com/nyattsu72/hanson-frontend-cource/pull/20/commits/1ee8e03e6dfe781319816401505fe06127f9fc99)